keyboards/montsinger/rebound/readme.md

```diff
@@ -5,11 +5,13 @@
 A conjoined-split 40% 4x12 ortholinear keyboard made and sold by Montsinger. [More info on Montsinger.net](https://montsinger.net).
 
 * Keyboard Maintainer: [Rossman360](https://github.com/rossman360)
-* Hardware Supported: Pro Micro, Elite-C
+* Hardware Supported: Rebound rev1 or rev2; Pro Micro or Elite-C
 * Hardware Availability: [Montsinger.net](https://montsinger.net)
 
 Make example for this keyboard (after setting up your build environment):
 
     make montsinger/rebound:default
```

Comment:
As currently implemented, the make command must specify either `/rev1` or `/rev2`; this syntax won't work.

```suggestion
```

Reply:
the change for line 8 is how my file already looked